docs/sidebarsGenAI.ts

```diff
@@ -46,18 +46,18 @@ const sidebarsGenAI: SidebarsConfig = {
       items: [
```

Comment:
Could we make the label 'Developer Workflow 👉' so that it looks consistent with other titles?